### DISCUSSION TITLE
OpenQuery for a Linked Server

### DISCUSSION THREAD

**Discussion by dinonappa1** [Azure/data-api-builder#2279] [Ideas]
I was just popping in to ask if there is a way to use OpenQuery(INSQL,"QUERY HERE") with this API builder? 
If not, then I was wondering if this would be something that could be added in?

**Comment by seantleonard** [comment]
Hi @dinonappa1, Thank you for your reaching out and I'm sorry for the delay in responding.  Are you expecting something more than adding `OpenQuery(INSQL,"QUERY HERE")` to a stored procedure and executing that stored procedure from DAB? If not, can you elaborate on your use case with more examples of how you'd use this and what you'd expect to be present in DAB's REST/GraphQL endpoints?
